models/allegro/join_matbench_preds.py

```
@@ -0,0 +1,109 @@
+
+dfs = {}
+for file_path in tqdm(files, desc="Loading results"):
+    if file_path in dfs:
+        continue
+    df_i = pd.read_json(file_path).set_index(Key.mat_id)
+    dfs[file_path] = df_i
+
+df_allegro = pd.concat(dfs.values()).round(4)
+
+if len(df_allegro) != len(df_wbm):  # make sure there is no missing structure
+    warnings.warn(
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)